### ISSUE TITLE
Logic of is_timestamp_complete

### ISSUE THREAD

**Issue by RCasatta** [opentimestamps/opentimestamps-client#28]
According to the [is_timestamp_complete](https://github.com/opentimestamps/opentimestamps-client/blob/master/otsclient/cmds.py#L211-L219) function you need one BitcoinBlockHeaderAttestation to consider the timestamp complete. However it could happen than another calendar server (in PendingAttestation) has a BitcoinBlockHeaderAttestation which has an eventually lower than the first height returned.
This information will never be asked to the calendar if I have a timestamp upgraded from the other calendar.
What about having an option in upgrade to consider a timestamp complete if all PendingAttestation are served?

**Comment by petertodd** [comment]
Yup, I think that makes sense - it's not complete until you have attestations for every pending attestation.

Equally, like I mentioned in 
https://github.com/opentimestamps/opentimestamps-client/issues/27, the upgrade command should *always* try to get new attestations even if the timestamp is complete: the calendar might be returning an even earlier timestamp than it previously did.
